kubernetes-client-api/src/main/java/io/fabric8/kubernetes/client/utils/KubernetesResourceUtil.java

```diff
@@ -444,7 +444,7 @@ public static Secret createDockerRegistrySecret(String dockerServer, String user
     String dockerConfigAsStr = Serialization.jsonMapper().writeValueAsString(dockerConfigMap);
 
     return new SecretBuilder()
-        .withNewMetadata().withName("harbor-secret").endMetadata()
+        .withNewMetadata().withName("docker-registry-secret").endMetadata()
```

Comment:
Let's make the secret name configurable. Could you please add another method parameter `secretName`

Reply:
@rohanKanojia  thx😄
could you merge this?